.env

```diff
@@ -7,7 +7,7 @@ STRIPE_PUBLISHABLE_KEY=pk_test_yndN9H0GcJffPe0W58Nm64cM00riYG4N46
 STORE_MAINTENANCE=false
 BADGR_URL=https://api.test.badgr.com
 TRUEABILITY_URL="https://app.trueability.com"
-CREDLY_URL="https://sandbox-api.credly.com/v1"
```

Comment:
Is working with Prod API a good idea on local? 
If not, leave the sandbox url here.

Reply:
Sandbox is not active anymore. Everything will have to be done on prod. So how we tackle that is for staging we have mapped to exam 4190 which is a staging exam and that gives out a staging badge. The production exam will give out a production badge